vars/main.yml

```diff
@@ -26,3 +27,7 @@ steamengine_play2_run_script: "startproject"
 # TOMCAT
 steamengine_tomcat_version: "{{ steamengine_tomcat_major_version }}.{{ steamengine_tomcat_minor_version }}"
 steamengine_tomcat_tomcat_url: "https://archive.apache.org/dist/tomcat/tomcat-{{ steamengine_tomcat_major_version }}/v{{ steamengine_tomcat_version }}/bin/apache-tomcat-{{ steamengine_tomcat_version }}.tar.gz"
+
+# DRUPAL
+steamengine_drupal_public_path: "{{ steamengine_project_root_path_web }}/{{ steamengine_drupal_public_directory }}"
+steamengine_drupal_persistent_path: "{{ steamengine_project_root_path }}/link"
```

Comment:
the project_root directory has been thought for code and is the home of any runtime deps of the project. This directory could be deleted, persitent storage should be placed in {{ steamengine_home_path }}/storage